CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Semantic versioning in our case means:
 - Forbids to use positional only `/` arguments
 - Adds `__call__` to list of methods that should be on top #1125
 - Now allows `_` to be used as a defined variable
+- Forbids to use raw strings `r''` when not necessary
```

Comment:
This is related to `0.15`, not `0.14`

Reply:
The header for 0.15 is not yet created. Should I create it or just remove the line of contribution?

Reply:
You can create it

Reply:
See 870f8bed7a6f92a55ec15e6e2e806d5344041f50